core/project.clj

```diff
@@ -1,11 +1,12 @@
-(defproject finagle-clojure/core "0.8.0-NUBANK"
+(defproject finagle-clojure/core "0.8.0-SNAPSHOT"
```

Comment:
This project is a fork so we added the suffix NUBANK to avoid conflicts between our version and the "original" version. The "original" version is deprecated and I talked with the main developer and he said that he don't have plans to work on it, but I still think that it's better for us to have something saying it explicitly. If it will break the default pipeline , maybe we can rename this project to nu-finagle-clojure. WDYT?

Reply:
In the `:release-tasks` section of the project.clj I added a step that removes the snapshot and adds `-NUBANK` at the end so we don't have to worry about doing it manually.